tests/test_pipelines_conversational.py

```diff
@@ -87,11 +88,7 @@ def test_integration_torch_conversation(self):
         self.assertEqual(len(conversation_1.past_user_inputs), 0)
         self.assertEqual(len(conversation_2.past_user_inputs), 0)
 
-        with self.assertLogs("transformers", level="WARNING") as log:
```

Comment:
Removed the warnings because it's tedious to manage, and they don't bring that much value to the test, the real test is the output of the conversation.